doc/source/tune/doc_code/key_concepts.py

```diff
@@ -73,15 +73,15 @@ def load_checkpoint(self, checkpoint_dir):
 # __config_start__
 config = {
     "uniform": tune.uniform(-5, -1),  # Uniform float between -5 and -1
-    "quniform": tune.quniform(3.2, 5.4, 0.2),  # Round to increments of 0.2
+    "quniform": tune.quniform(3.2, 5.4, 0.2),  # Round to multiples of 0.2
     "loguniform": tune.loguniform(1e-4, 1e-1),  # Uniform float in log space
-    "qloguniform": tune.qloguniform(1e-4, 1e-1, 5e-5),  # Round to increments of 0.00005
+    "qloguniform": tune.qloguniform(1e-4, 1e-1, 5e-5),  # Round to multiples of 0.00005
     "randn": tune.randn(10, 2),  # Normal distribution with mean 10 and sd 2
-    "qrandn": tune.qrandn(10, 2, 0.2),  # Round to increments of 0.2
+    "qrandn": tune.qrandn(10, 2, 0.2),  # Round to multiples of 0.2
     "randint": tune.randint(-9, 15),  # Random integer between -9 and 15
-    "qrandint": tune.qrandint(-21, 12, 3),  # Round to increments of 3 (includes 12)
+    "qrandint": tune.qrandint(-21, 12, 3),  # Round to multiples of 3 (includes 12)
     "lograndint": tune.lograndint(1, 10),  # Random integer in log space
-    "qlograndint": tune.qlograndint(1, 10, 2),  # Round to increments of 2
+    "qlograndint": tune.qlograndint(1, 10, 2),  # Round to multiples of 2
```

Comment:
I was wondering maybe an example easier to understand than wording?
e.g. the qrandint(1, 10, 3) translates to choosing randomly among [3, 6, 9] as we were discussing yesterday?

Reply:
It should choose between [1, 4, 7, 10] though, right?

Reply:
The quantized sampling rounds the sampled values to multiples of q rather than integer increment of q from the lower bound. So `qrandint(1,10,3)` will randomly sample from `[3,6,9]` rather than `[1,4,7,10]`

Reply:
@ilee300a do you want to provide more details as to why the decision was made so? My understanding is there is some impl that is shared with other sampling methods, which may require [3, 6, 9] instead of [1, 4, 7, 10].

Reply:
When using other libraries, the expected behavior of quantization is to select those that are divisible by the given q within the bounds. For example, when using BOHB, it expects the provided lower and upper bound to be a multiple of q and provides multiples of q within those bounds as the samples. So if we run `qrandint(1,10,3)`, it actually samples multiples of 3 between 3 and 9(inclusive).

Reply:
Makes me wonder if we should raise an exception during domain initialisation if the bounds are not divisible instead.